src/main/webapp/Renderer.js

```diff
@@ -0,0 +1,22 @@
+/*
+** Class for adding the events stored in the calendar to the html page
+*/
+export default class Renderer {
+    constructor(eventList) {
+        this.eventList = eventList;        
+    }
+
+    render(events) {
+        this.eventList.innerHTML = "";
+
+        for (var index in events.list) {
```

Comment:
Prefer to use const/let instead of var (those are more stable and are the standard for modern JavaScript).
When you have a variable that changes, use let.
When you have a constant, use const.
In this case, index doesn't change!

Please apply throughout!

Reply:
Understood.